Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
> Hence the word "contractor", maybe ?

:sweat_smile: 

> Should we make it private then?

ATM I can't see a place where `fixFieldDescription` would be called outside a `XXXBuilderInterface` instance.
So AFAICU right now, I'd say we should put `fixFieldDescription` on `FormContractorInterface` and make it private to the other builders. And also remove the `BuilderInterface` since it does nothing, and having `FormContractorInterface` inherit from it is misleading.

Then we can properly document why it is needed to have `FormContractorInterface::fixFieldDescription` and `FormContractorInterface::getDefaultOptions`: because we don't want to inherit from the Symfony FormBuilder in the persistence bundles, so we add a small "persistence-related" middleware between the FormMapper and the FormBuilder.

Also, since the `fixFieldDescription` implementations seem to always re-populate the `FieldDescription` instance with the `ModelManager` metadata, maybe the `FieldDescription` construction should actually be the responsibility of the Builders ? In which case, and if it is true that a `FieldDescription` is instanciated only by a builder, we could also consider the metadata of `FieldDescription` instances immutable and let these objects construct themselves from the metadata. And consequently remove some of the setters in `FieldDescriptionInterface`, like `setAssociationMapping`, `setFieldMapping`, `setParentAssociationMappings` et al. ?